packages/workflow/src/workflow.ts

```diff
@@ -915,24 +915,21 @@ export function makeContinueAsNewFunc<F extends Workflow>(
 }
 
 /**
- * Continues current Workflow execution as new with default options.
+ * [Continues-As-New](https://docs.temporal.io/concepts/what-is-continue-as-new/) the current Workflow Execution with
+ * default options.
  *
- * Shorthand for `makeContinueAsNewFunc<F>()(...args)`.
+ * Shorthand for {@link makeContinueAsNewFunc}.
  *
  * @example
  *
- * ```ts
- * import { continueAsNew } from '@temporalio/workflow';
+ *```ts
+ *import { continueAsNew } from '@temporalio/workflow';
  *
- * export function myWorkflow(n: number) {
```

Comment:
snipsync would have found this (but the example wouldn't show up in IDEs).
There's probably a better solution for testing the examples but it's not critical for this PR.